docs/source/contributing.rst

```diff
@@ -0,0 +1,11 @@
+Contributing
+------------
+
+PyKE is being actively developed on its `GitHub repository <https://github.com/KeplerGO/PyKE>`_.
+
+If you have found a bug on PyKE, please open an issue on our GitHub page or email us at
+keplergo@mail.arc.nasa.gov. We also welcome our users to suggest enhancements or new tools for PyKE.
+
+If you would like to contribute with a bug fix or a new tool, please go ahead and open a pull request.
+If in doubt on how to open a pull request, we recommend you to check the tutorial
+`How to make a code contribution by Astropy <http://docs.astropy.org/en/stable/development/workflow/development_workflow.html>`_.
```

Comment:
Great!  I'd just add one extra sentence here encouraging folks to e-mail us early, or open an issue, to check if a new feature seems like a good idea.